src/routes/blog/post/How-to-put-privacy-first/+page.markdoc

```diff
@@ -0,0 +1,86 @@
+
+But there is more to gain than just privacy-friendly data.
+
+## Building experience and trust with the developer community
+
+As we often repeat, we are heavily focused on the developer experience. It is at the core of everything we do. Considering this, there are several reasons why we decided to remove most of the tags, pixels, and cookies from our website:
+
+- **Privacy:** As many studies show, people want to [control the information they share](https://venturebeat.com/data-infrastructure/report-94-of-consumers-want-control-over-the-information-they-share-with-companies/) with other parties online. By removing most of the tags, pixels, and cookies, we reduce the amount of personal data we collect from the Appwrite website and Console. And as we focus on collecting first-party data, we grant you that control.
+- **Performance:** Tags, pixels, and cookies can slow down a website's loading time. By removing them, we improve the performance of our website and the Console for you. And a [study from Akamai](https://www.akamai.com/newsroom/press-release/akamai-releases-spring-2017-state-of-online-retail-performance-report), shows that a two-second delay in web page load time increases bounce rates by 103 percent. So, improving performance is worth it.
+- **Security:** This tracking technology can also track you across the web and collect intrusive data about you. By removing them, we are making our website more secure for you and preventing potential [data breaches](https://venturebeat.com/security/report-multiple-data-breaches-common-in-past-year/).
+
+Overall, it made much sense to go down this route and focus on delivering a better experience and enhancing trust with the community. And I think it’s fair to say that with the removal of marketing pixels, cookies, and tags, we didn’t lose anything; we merely gained.
+
+## Building a privacy-friendly and open-source growth stack
+
+We still have the necessary tools to improve our product, the developer experience, and analyze our website traffic. That said, we will continue to explore the possibilities of moving towards a fully open-source and privacy-friendly growth tech stack. Not only do we feel it is the right thing to do, but it is also rooted in our culture. We want to treat you as we want to be treated by any vendors we use. For now, this has been the most sensible decision we could have made for Appwrite and the community. We are excited to take privacy further and further, so who knows where we will stand a year from now.
```

Comment:
`We want to treat you as we want to be treated by any vendors we use`

Powerful sentence, I think it should be used in the beginning of the article. It makes it very clear we understand the pain point, and have been on receiving end too.